cmake/DftbPlusUtils.cmake

```diff
@@ -223,6 +223,10 @@ function (dftbp_ensure_config_consistency)
 
   endif()
 
+  # Check minimal compiler versions
+  set(fortran_minimal_versions "GNU;9.0" "Intel;19.0" "NAG;7.0")
```

Comment:
Does this work reliably with CMake using the semicolon as default delimiter? Or does it just assume that we will later always remove the entries of the list in pairs?

Reply:
Yes, the routine doing the check pops always 2 entries at a time. One could equivalently write it as `set(... "GNU" "9.0" "Intel" "19.0")`, but I've found the form above considerably easier to read.